src/test/java/com/mycompany/app/service/BasicStockServiceTest.java

```diff
@@ -0,0 +1,122 @@
+package com.mycompany.app.service;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.*;
+
+
+/**
+ * JUnit tests for the BasicStockService 
+ *
+ * @author Linda M Roseberry
+ */
+public class BasicStockServiceTest {
```

Comment:
did a little too much work here, since basic stock service is supposed to be just hard coded 'dummy' code. But good effort.